docs/source/conf.py

```diff
@@ -240,19 +248,25 @@
 # (source start file, target name, title, author,
 #  dir menu entry, description, category)
 texinfo_documents = [
-  ('index', 'netengine', u'netengine Documentation',
-   u'Alessandro Bucciarelli, Federico Capoano', 'netengine', 'One line description of project.',
-   'Miscellaneous'),
+    (
+        'index',
+        'netengine',
+        u'netengine Documentation',
+        u'Alessandro Bucciarelli, Federico Capoano',
+        'netengine',
+        'One line description of project.',
+        'Miscellaneous',
```

Comment:
shall we update this since we are here or shall we do this later?

Reply:
I think it may be better to do this in https://github.com/openwisp/netengine/issues/56

Reply:
Ok for #56